src/main/java/uk/gov/companieshouse/charges/data/config/ExceptionHandlerConfig.java

```diff
@@ -1,14 +1,16 @@
 package uk.gov.companieshouse.charges.data.config;
 
+import java.io.IOException;
 import java.time.LocalDateTime;
-import java.util.LinkedHashMap;
-import java.util.Map;
-import java.util.UUID;
+import java.util.*;
```

Comment:
this will fail check style. Could you please run the maven build and check locally?